### ISSUE TITLE
Shutting down configurator will not automatically shut down other containers

### ISSUE THREAD

**Issue by alan791205** [oharastream/ohara#1041]
As title, whenever configurator is shutted down, other containers will stay alive, 
but the next time configurator is executed, getting infos of zookeeper and brokers will not return any of those alive containers. So I might thought there is no zookeeper or broker.
But when I added new zookeepers or brokers with same infos will return 
`"message": "zookeeper cluster:zk00 is running"`
Will shutting down everything be a feature or the behavior now is how this is expected?

**Comment by chia7712** [comment]
Shutting down configurator doesn't **release** all resources since we don't want to make a **unexpected** data lose. Also, it opens a door to restart configurator with all previous data (and services).

At any rate, it is ok to add a **flag** to do kill all in shutdown. @alan791205 any idea?

**Comment by alan791205** [comment]
> At any rate, it is ok to add a flag to do kill all in shutdown.

+1 to this. 
But I did not see how to restart configurator with all previous data, is there a flag when starting configurator? 
I saw [this](https://github.com/oharastream/ohara/blob/master/docs/user_guide.md#execute-configurator) on the node section and I have tried to restart it by using that param but has no effect on starting with previous data, is this still a ongoing thing or am I missing something?
